jobs/release.Jenkinsfile

```diff
@@ -307,13 +307,21 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
 
                 // make AMIs public if not already the case
                 if (!pipecfg.clouds?.aws?.public) {
-                    shwrap("""
+                    def rc = shwrapRc("""
                     cosa shell -- plume make-amis-public \
                         --version ${params.VERSION} \
                         --stream ${params.STREAM} \
                         --bucket-prefix ${bucket_prefix} \
                         --aws-credentials \${AWS_BUILD_UPLOAD_CONFIG}
                     """)
+                    // see https://github.com/coreos/coreos-assembler/pull/3277
+                    if (rc == 77) {
+                        // Display a warning, but continue.
+                        def message = "Failed to make AMIs public in some regions"
+                        warnError(message: message) {
+                            error(message)
+                        }
+                    }
```

Comment:
I feel like we're missing a case here. For example, what if the rc isn't `77` or `0`? We want to `error()` in that case.